### PR TITLE
Fix Broken Links to Ingress Guide and Add Prometheus & Alertmanager Exposure Guide

### PR DESCRIPTION
**Pull request by Dharma-09**
## Description

Updates all broken links to the Ingress Guide throughout the website.
Replaces non-functional links with a working reference to the [Prometheus & Alertmanager Exposure Guide](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/exposing-prometheus-and-alertmanager.md).

## Issues
#6978
https://github.com/prometheus-operator/website/issues/116

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [X] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
NONE
```

### CODE REVIEW

**Comment by nicolastakashi** [comment]
cc @AshwinSriram11

**Review by AshwinSriram11 — COMMENTED** [review]
Sorry for the late review @Dharma-09. LGTM :+1: Just to make sure that everything works well, can you make another PR in the website repo to show the website preview?

**Comment by Dharma-09** [comment]
Thanks for the review @AshwinSriram11  👍 I’ll create a follow-up PR in the website repo.

**Comment by AshwinSriram11** [comment]
> I don't think you need to add these new changes you made in the recent commit. What I meant was to change the source path in the synchronization.sh file in the website repo like I mentioned in that commit.
> 
> Look at this example: [prometheus-operator/website@e71a061#:~:text=git%20clone%20https,repos/kube%2Dprometheus](https://github.com/prometheus-operator/website/commit/e71a0614f39bfc074deb6270666f8c48b6cec5ed#:~:text=git%20clone%20https,repos/kube%2Dprometheus)

When the PR in the main repo was merged, I changed it back to the original source path. This way, you can check whatever changes you make in the docs are working neatly on the website.

**Comment by Dharma-09** [comment]
I was a bit confused earlier. Thanks for clarifying!

**Comment by AshwinSriram11** [comment]
Can you revert to commit with only changes to docs? We don't want any changes other than what we had before.

File: Documentation/platform-guide.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ draft: false
 description: Getting started page for Platform Guide
 ---
 
-This guide assumes you have a basic understanding of the Prometheus Operator. If you are new to it, please start with the [Introduction](introduction.md) page before proceeding. This guide will walk you through deploying Prometheus and Alertmanager instances.
+This guide assumes you have a basic understanding of the Prometheus Operator. If you are new to it, please start with the [Introduction]({{< ref "introduction.md" >}}) page before proceeding. This guide will walk you through deploying Prometheus and Alertmanager instances.
```
Comment (AshwinSriram11) [inline Documentation/platform-guide.md:14]:
```suggestion
This guide assumes you have a basic understanding of the Prometheus Operator. If you are new to it, please start with the [Introduction](introduction.md) page before proceeding. This guide will walk you through deploying Prometheus and Alertmanager instances.
```

File: Documentation/platform-guide.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ spec:
 Once the Service is created, the Alertmanager web server is available under the
 node's IP address on port `30903`.
 
-> Note: Exposing the Alertmanager web server this way may not be an applicable solution. Read more about the possible options in the [Ingress guide](user-guides/exposing-prometheus-and-alertmanager.md).
+> Note: Exposing the Alertmanager web server this way may not be an applicable solution. Read more about the possible options in the [Ingress guide]({{< ref "user-guides/exposing-prometheus-and-alertmanager.md" >>}}).
```
Comment (AshwinSriram11) [inline Documentation/platform-guide.md:153]:
```suggestion
> Note: Exposing the Alertmanager web server this way may not be an applicable solution. Read more about the possible options in the [Ingress guide]({{< ref "user-guides/exposing-prometheus-and-alertmanager.md" >}}).
```
Just this last change and we are probably done.

**Comment by AshwinSriram11** [comment]
LGTM :+1: Just made a commit in the website repo and the preview works fine

**Comment by AshwinSriram11** [comment]
Let's wait for all the tests to pass and merge this PR and then you can change the path in the synchronization file to the original path and we are done.

**Review by AshwinSriram11 — APPROVED** [review]

**Comment by Dharma-09** [comment]
Thank you, @AshwinSriram11, for your help!

**Review by nicolastakashi — APPROVED** [review]

**Pull request merged**